ui/src/views/Settings.vue

```diff
@@ -148,27 +148,52 @@
             >{{$t('advanced_mode')}}</a>
         </legend>
 
-        <div v-if="configuration.status && configuration.advanced && configuration.Recidive_Perpetual" 
-          class="alert alert-info alert-dismissable">
-          <span class="pficon pficon-info"></span>
-          <strong>{{$t('settings.recidive_perpetual_label')}}</strong>&#32;
-          {{$t('settings.recidive_perpetual_info')}}
-        </div>
-
         <div
           v-if="configuration.status && configuration.advanced"
-          :class="['form-group', errors.Recidive_Perpetual.hasError ? 'has-error' : '']"
+          :class="['form-group', errors.RecidiveBan.hasError ? 'has-error' : '']"
         >
           <label
             class="col-sm-2 control-label"
             for="textInput-modal-markup"
-          >{{$t('settings.Recidive_Perpetual')}}</label>
+          >{{$t('settings.Recidive')}}
+          <doc-info
+            :placement="'top'"
+            :title="$t('settings.Recidive')"
+            :chapter="'Recidive'"
+            :inline="true"
+          ></doc-info>
+          </label>
           <div class="col-sm-5">
-            <input type="checkbox" v-model="configuration.Recidive_Perpetual" class="form-control">
+              <input type="radio" id="IncrementalBanTime" value="incremental"
+                v-model="configuration.RecidiveBan" class="form-control" >
+              <label for="IncrementalBanTime">{{$t('settings.Recidive_Incremental')}}
+                <doc-info
+                :placement="'top'"
+                :title="$t('settings.Recidive_Incremental')"
+                :chapter="'Incremental_BanTime'"
+                :inline="true"
+                ></doc-info>
+              </label>
+              <br />
+              <input type="radio" id="RecidiveBanTime" value="static"
+                v-model="configuration.RecidiveBan" class="form-control" >
+              <label for="RecidiveBanTime">{{$t('settings.Recidive_BanTime')}}
+                <doc-info
+                :placement="'top'"
+                :title="$t('settings.Recidive_BanTime')"
+                :chapter="'Recidive_BanTime'"
+                :inline="true"
+                ></doc-info>
+              </label>
+              <br />
+              <input type="radio" id="RecidiveBanTime" value="disabled"
```

Comment:
The radio button doesn't behave correctly: if you click on 'Disabled' label, the associated radio is not enabled.

```suggestion
              <input type="radio" id="RecidiveBanDisabled" value="disabled"
```